packages/core/src/physics/PhysicsManager.ts

```diff
@@ -232,7 +264,19 @@ export class PhysicsManager {
    * @internal
    */
   _update(deltaTime: number): void {
-    this._nativePhysicsManager.update(deltaTime);
+    const {
+      _fixedTimeStep: fixedTimeStep,
+      _nativePhysicsManager: nativePhysicsManager
+    } = this;
+    const componentManager = this._engine._componentsManager;
+
+    const simulateTime = deltaTime + this._restTime;
+    const step = Math.min(this._maxStepCount, Math.floor(simulateTime / fixedTimeStep));
```

Comment:
Maybe `const step = Math.floor(Math.min(this._maxSumTimeStep, simulateTime) / fixedTimeStep);` is better and can save a global field `this._maxStepCount`

Reply:
+1